initfiles/bash/etc/init.d/hpcc-init.in

```diff
@@ -36,8 +36,11 @@
 ### BEGIN INIT INFO
 # chkconfig: 235 40 90
 # Description: Starts the HPCC processes
+# Short-Description: Controls HPCC Processes
 # Provides: hpcc-init
 # Required-Start: dafilesrv
+# Default-Start: 2 3 4 5
```

Comment:
Can you leave this blank ?  Is that acceptable ??
I propose dafilesrv starts on reboot but hpcc-init does not.